pkgs/development/tools/mysql-shell/default.nix

```diff
@@ -0,0 +1,174 @@
+{ lib
+, stdenv
+, pkg-config
+, cmake
+, fetchurl
+, git
+, bison
+, openssl
+, protobuf
+, curl
+, zlib
+, libssh
+, zstd
+, lz4
+, boost
+, readline
+, libtirpc
+, rpcsvc-proto
+, libedit
+, libevent
+, icu
+, re2
+, ncurses
+, libfido2
+, v8
+, python
+, cyrus_sasl
+, openldap
+, numactl
+, cctools
+, CoreServices
+, developer_cmds
+, DarwinTools
+, testVersion
+, mysql-shell
+}:
+
+with builtins;
+let
+  pythonDeps = [
+    python.pkgs.certifi
+    python.pkgs.paramiko
+  ];
+  site = ''
+
```

Comment:
```suggestion
```

Reply:
It's on purpose. It appends `site` content to a python file.